### DISCUSSION TITLE
Pre-release 0.3.1

### DISCUSSION THREAD

**Discussion by IsaacMarovitz** [Whisky-App/Whisky#128] [Announcements]
**Bug Fixes:**
- Fixed GPTK Installation

**Full Changelog**: https://github.com/IsaacMarovitz/Whisky/compare/pre-0.3.0...pre-0.3.1

<hr /><em>This discussion was created from the release <a href='https://github.com/IsaacMarovitz/Whisky/releases/tag/pre-0.3.1'>Pre-release 0.3.1</a>.</em>

**Comment by GaryL87** [comment]
The battle net launcher shows two popups
1. You need at least Windows 10(version 1909).
2. Unsupported OS Version (10.0.18362).

**Reply by GaryL87** [reply]
it might be related to the brew configuration.
the return of "which brew" is /opt/homebrew/bin/brew

**Reply by IsaacMarovitz** [reply]
It is not. You need to change your Windows build settings in your Bottle's config

**Reply by GaryL87** [reply]
The windows setting is already 10 and the Build Version is 18362.

**Reply by vladashram** [reply]
Change your build to 19042.

**Reply by GaryL87** [reply]
@vladashram How can I change the build to 19042?

**Reply by GaryL87** [reply]
Even if If entered 19042 to Build Version, the battle net launcher failed to launch the diablo 4. 
When I hit the Play button the previous pop up messages are gone, but the Play button changed to Launching, Playing Now and back to Updating and Play. nothing happens.

**Comment by ariofrio** [comment]
Correction to the installation instructions: you must follow the following first two steps from the Read Me.rtf of the Game Porting Toolkit before Whisky will work:

> - Ensure the Command Line Tools for Xcode 15 beta are installed. Visit https://developer.apple.com/downloads to download these tools.
> - Open Terminal.
> - The Game Porting Toolkit runs under Rosetta 2. Ensure that Rosetta 2 is installed.
> ```
> softwareupdate --install-rosetta
> ```

If you haven't done both of these things (for example if this is a fresh macOS install), then Whisky will seem to succeed, but the Programs list will be empty, the Build Version will show a spinner indefinitely, and Open Control Panel / Open Registry Editor / Open Wine Configuration will do nothing.

It would be good to both add these instructions to the README and show some kind of error message or do these steps automatically, if they haven't been done yet, inside the app.

**Reply by IsaacMarovitz** [reply]
Rosetta installation is coming in the next release

**Reply by ariofrio** [reply]
Great! FYI, I just noticed that I did not check whether Whisky would work with only Rosetta installed, but not Command Line Tools for Xcode. Presumably it's only needed for building Game Porting Toolkit, which Whisky pre-builds.
